OracleOGG/Oracle/README.md

```diff
@@ -0,0 +1,73 @@
+Oracle GoldenGate for Oracle on Docker
+===============
+Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. 
+
+For more information about Oracle GoldenGate 12c please see the Oracle GoldenGate Online Documentation at http://docs.oracle.com/goldengate/c1221/gg-winux/index.html.
+
+For more information about Oracle Database 12c, please see the Oracle Database Online Documentaion at http://docs.oracle.com/en/database/index.html.
+
+## How to build Oracle GoldenGate for Oracle on Docker
+This project offers sample Dockerfile and other associated file for this build:
+ * V100692-01.zip (Oracle GoldenGate 12c for Oracle (12.2.0.1))
+ * database1.zip (Oracle Database 12c Release 1 (12.1.0.2) Enterprise Edition (Disk 1))
+ * database1.zip (Oracle Database 12c Release 1 (12.1.0.2) Enterprise Edition (Disk 1))
+ * runSQL.sql - SQL file to run required steps
+ * Dockerfile - Build file for Docker image
+ * entrypoint.sh - steps required to install Oracle Database 12c and Oracle GoldenGate 12c
+ * db_install.rsp - file to install Oracle Database 12c silently
+
+To rebuild the Oracle GoldenGate for Oracle Docker image, use the following command:
+
+    docker build -t oggora:12.2.0.1.1 -f Dockerfile .
+ 
+## Running Oracle GoldenGate for Oracle in a Docker container
+To run your Oracle Database Docker image use the **docker run** command as follows:
+
+    docker run -d --privileged --name oggora \
```

Comment:
We will not accept any image that requires `--privileged` mode. Please drop this requirement.